tweakable_encryptor_test.go

```diff
@@ -6,6 +6,7 @@ package saltpack
 import (
 	"bytes"
 	"crypto/sha512"
+	"fmt"
```

Comment:
My preference in the past has been to put debugging hooks in the main encryptor, to avoid having to maintain both copies. But @maxtaco has worried that there'd be too much risk of a debugging hook getting called in prod. Maybe we could compromise on [some way to gate the hooks to testing mode only](http://stackoverflow.com/a/36666114/823869)?

On the signcryption side of things, I was able to get by without needing debug hooks at all, but partially msgpack-decoding a couple different things in the output and tweaking them manually. Honestly though, I only did that because I wanted to avoid all the duplication that a second encryptor would require, and the tests would've been easier to understand with proper hooks.